kubernetes/core/src/main/kotlin/com/google/kubernetes/tools/core/KubectlCommandLineState.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.kubernetes.tools.core
+
+
+import com.intellij.execution.ExecutionException
+import com.intellij.execution.Executor
+import com.intellij.execution.configurations.*
+import com.intellij.execution.process.KillableProcessHandler
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.options.SettingsEditor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.guessProjectDir
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.util.xmlb.XmlSerializer
+import org.jdom.Element
+import java.io.File
+import javax.swing.JComponent
+
+/**
+ * Runs Skaffold on command line in both "run"/"dev" modes based on the given
+ * [AbstractSkaffoldRunConfiguration] and current IDE project. [startProcess] checks configuration
+ * and constructs command line to launch Skaffold process. Base class manages console
+ * window output and graceful process shutdown (also see [KillableProcessHandler])
+ *
+ * @param environment Execution environment provided by IDE
+ * @param executionMode Skaffold  mode execution, i.e. DEV
+ */
+class KubectlCommandLineState(
```

Comment:
We don't need a `CommandLineState` yet for kubectl. `CommandLineState` is for setting up a console window to display the execution of the process (think of the skaffold process window that shows up when your run a deployment). In this case, we are first interested in just executing kubectl and getting the results (not actually showing a console window with the command output).